data/map.txt

```diff
@@ -28487,6 +28713,15 @@ planet Oblivion
 		threshold 3000
 		fleet "Large Militia" 8
 
+planet Ochrescoop
+	attributes station hai
+	landscape land/space6
+	description `A massive plaque attached to a bulkhead in the promenade entry proudly denotes this station to be run by "Yoot Ki Magnetic Fuel Harvesting". It seems that the powerful magnetic field of the gas giant below makes this station an ideal location for harvesting, and re-harvesting, the massive fuel volumes demanded by the Hai space industry. Despite a plaque also proudly denoting nearly 75,000 years of operation, the station seems to be in an excellent state of repair.`
+	spaceport `The spaceport, which consists of a wide promenade running around the interior of the station, seems to be divided into two distinct areas: the lower level, which contains all manner of amenities, services, and food places, and the upper level, which hosts the official Yoot Ki sales center in addition to a suite of different facilities for various research institutes. As it turns out, the powerful magnetic field of the world below also has certain advantages for scientific experiments.`
+	spaceport `	Floating next to the stairs a holographic sign proudly states that it's been over 7000 years since the last fatal safety accident.`
```

Comment:
```suggestion
	spaceport `	A holographic sign floating next to the stairs proudly states that over 7,000 years have elapsed since the past fatal safety accident.`
```